### ISSUE TITLE
 Looking for help with upgrading to specific version of Symfony

### ISSUE THREAD

**Issue by givihuda** [umpirsky/Symfony-Upgrade-Fixer#44]
I was hoping this could help me with a Symfony 2.8 to 3.4 upgrade, then later for a 3.4 to 4.x upgrade, but it seems that it only compares my Symfony version with the latest... which isn't helpful right now.

I'm thinking maybe there's a commit that will check Symfony version 3.4, but I'm not seeing any indication as to where.

Also, the README doesn't specify which OS and version its CLI commands are for. I'm on macOS Big Sur, and Apple has changed how things work significantly between versions to a surprising degree.

**Comment by umpirsky** [comment]
Only fixers available are https://github.com/umpirsky/Symfony-Upgrade-Fixer#fixers-available, unfortunately I don't have much time for this project, that's why it's not finished.

**Comment by givihuda** [comment]
Right, I saw those, but my question was about version number differences.

So, there's no commits I can reference to check that contain comparisons against Symfony versions that aren't the latest version?

**Comment by umpirsky** [comment]
Unfortunately no. It was useful to upgrade from Symfony 2 to Symfony 3.
